ui-src/src/app/configurations/configurations.component.html

```diff
@@ -1,10 +1,22 @@
 <div class="container">
+    <div class="row">
+        <div class=" column col-xs-7 col-sm-7 col-md-7 col-lg-7 ">
+            <label>Config File Path *</label>
+            <input class="form-control" type="string" [(ngModel)]="configPath" id="config-path" required>
+        </div>
+        <div class=" column col-xs-4 col-sm-4 col-md-4 col-lg-4 ">
```

Comment:
Just curious have we checked this on a mobile view?.

Reply:
No you don't, I was just wondering about all the column sizes. I think you can simply get way with one. `col-md-4`